work_queue/src/work_queue_process.c

```diff
@@ -229,28 +231,58 @@ static char * invoke_coprocess_function(char *input) {
 		fatal("could not lookup address of localhost");
 	}
 
-	struct datagram *dgram = datagram_create(DATAGRAM_PORT_ANY);
-	if(!dgram) {
-		fatal("could not create datagram: %s", strerror(errno));
+	timestamp_t curr_time = timestamp_get();
+	time_t stoptime = curr_time + timeout;
+
+	bool connected = false;
```

Comment:
Let's change this one to  `int connected = 0`, and use `1` for true, to make it uniform with the rest of the code.

Reply:
Done!